### PR TITLE
Coverity fallout

### PR DESCRIPTION
**Pull request by mmichal10**

### CODE REVIEW

File: modules/cas_cache/layer_cache_management.c [hunk 1]
```diff
@@ -1060,7 +1060,7 @@ int cache_mngt_prepare_cache_cfg(struct ocf_mngt_cache_config *cfg,
 	memset(device_cfg, 0, sizeof(*device_cfg));
 	memset(atomic_params, 0, sizeof(*atomic_params));
 
-	strncpy(cfg->name, cache_name, OCF_CACHE_NAME_SIZE);
```
Comment (micrakow) [inline modules/cas_cache/layer_cache_management.c]: please add some spaces

File: casadm/statistics_view_text.c [hunk 2]
```diff
@@ -882,6 +882,11 @@ static int adjust_column_widths(struct view_t *this,
 
 	for (i = 0 ; i != w ; ++i) {
 		int this_width = vector_get(&prv->col_w, i);
+		/*
+		 * This conditin is exactly the same as in above loop, where
```
Comment (arutk) [inline casadm/statistics_view_text.c]: *conditin 
the word 'where' is repeated twice

File: casadm/statistics_view_text.c [hunk 3]
```diff
@@ -882,6 +882,11 @@ static int adjust_column_widths(struct view_t *this,
...
+		 * where above_avg_cols is incremented. So there is no risk of division
```
Comment (micrakow) [inline casadm/statistics_view_text.c]: double 'where'

File: casadm/cas_lib.c [hunk 4]
```diff
@@ -597,7 +597,7 @@ static int get_core_device(int cache_id, int core_id, struct core_device *core)
 
 	core->id = core_id;
 	core->cache_id = cache_id;
-	strncpy_s(core->path, sizeof(core->path), cmd_info.core_path_name,
+	strncpy_s(core->path, sizeof(core->path)-1, cmd_info.core_path_name,
```
Comment (micrakow) [inline casadm/cas_lib.c]: plase add some spaces

Comment (arutk) [inline casadm/cas_lib.c]: According to strncpy_s comment header the second param is:

dmax      - restricted maximum length of the resulting dest, including the null

... so adding -1 here doesn't seem to be correct.

**Comment by mmichal10** [comment]
retest this please

**Pull request merged**